packages/blade/src/components/ActionList/ActionListFooter.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import styled from 'styled-components';
+import { componentIds } from './componentIds';
+import Box from '~components/Box';
+import type { IconComponent } from '~components/Icons';
+import { useDropdown } from '~components/Dropdown/useDropdown';
+import { isReactNative, makeSize } from '~utils';
+import { BaseText } from '~components/Typography/BaseText';
+
+type ActionListFooterProps = {
+  title?: string;
+  leading?: React.ReactNode;
+  trailing?: React.ReactNode;
+};
+
+const StyledActionListFooter = styled(Box)((props) => {
+  return {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    padding: `${makeSize(props.theme.spacing[3])} ${makeSize(props.theme.spacing[5])}`,
+    backgroundColor: props.theme.colors.brand.gray.a50.lowContrast,
+    margin: `${makeSize(props.theme.spacing[3])} -${makeSize(props.theme.spacing[3])} -${makeSize(
+      props.theme.spacing[3],
+    )}`,
+  };
+});
+
+const ActionListFooter = (props: ActionListFooterProps): JSX.Element => {
+  const footerRef = React.useRef<HTMLDivElement | null>(null);
+  const {
+    setShouldIgnoreBlur,
+    setHasFooterAction,
+    onSelectKeydown,
+    activeIndex,
+    setIsOpen,
+  } = useDropdown();
+
+  React.useEffect(() => {
+    if (!isReactNative() && footerRef.current?.querySelector('button, a')) {
+      setHasFooterAction(true);
+    }
+  }, [setHasFooterAction, props.trailing]);
+
+  const isOnlyActionButton = !props.title && !props.leading && props.trailing;
+
+  return (
+    <StyledActionListFooter
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      ref={footerRef as any}
+      // eslint-disable-next-line
+      // @ts-ignore: Ignoring because the TS fails for React Native and works for web
+      onMouseDown={() => {
+        setShouldIgnoreBlur(true);
+      }}
+      // eslint-disable-next-line
+      // @ts-ignore: Ignoring because the TS fails for React Native and works for web
+      onKeyDown={(e) => {
+        const nativeEvent = e.nativeEvent;
+        // We ignore the selection keydowns on footer to let users click on items
+        if ((nativeEvent.key === ' ' || nativeEvent.key === 'Enter') && activeIndex < 0) {
+          return;
+        }
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        onSelectKeydown?.({ event: e.nativeEvent } as any);
+      }}
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
+      // @ts-ignore: Ignoring because the TS fails for React Native and works for web
+      onBlur={(e) => {
+        const nextItem = e.nativeEvent.relatedTarget;
```

Comment:
Why use `e.nativeEvent.relatedTarget` instead of `e.relatedTarget` ?? That will solve the `getAttribute` not present issue too.

Reply:
oh yes right!! changed it in upcoming commit